App.js

```diff
@@ -1,13 +1,81 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, TextInput, Pressable } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 
-export default function App() {
+import Button from './src/components/Button';
+import Input from './src/components/Input';
+
+
+function Home () {
+  return (
+    <View style={styles.container}>
+      <Text>Please enter credentials</Text>
+      <Input placeholder="Username" />
+      <Input placeholder="Password" secureTextEntry={true} />
```

Comment:
Any property is `true` by default. So you can shortify it: 
`<Input placeholder="Password" secureTextEntry />`